.github/workflows/release.yaml

```diff
@@ -26,6 +57,7 @@ jobs:
     needs: [build-packages, build-packages-arm64]
     uses: falcosecurity/falco/.github/workflows/reusable_publish_packages.yaml@master
     with:
+      bucket_suffix: ${{ steps.get_settings.outputs.bucket_suffix }}
```

Comment:
We need to add the `needs` on the `release-settings` job so that it is always performed before this job; then we need to use
```
${{ needs.release-settings.outputs.bucket_suffix }}
```
And same for other usages of `steps.get_settings` :)

Reply:
That was a great catch.